.gitignore

```diff
@@ -0,0 +1 @@
+.DS_STORE
```

Comment:
OS or development-environment-specific things should go in your global ignore file

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore